store/postgres/src/store.rs

```diff
@@ -118,16 +119,20 @@ pub struct StoreConfig {
 }
 
 /// Commonly needed information about a subgraph that we cache in
-/// `Store.subgraph_cache`. Only immutable subgraph data can be cached this
+/// `Store.schema_cache`. Only immutable subgraph data can be cached this
 /// way as the cache lives for the lifetime of the `Store` object
 #[derive(Clone)]
-struct SubgraphInfo {
+struct SchemaCacheEntry {
     /// The schema as supplied by the user
     input: Arc<Schema>,
     /// The schema we derive from `input` with `graphql::schema::api::api_schema`
     api: Arc<Schema>,
     /// The name of the network from which the subgraph is syncing
     network: Option<String>,
+    /// The imported schemas which are referenced by name or were not available to merge
+    unstable_imports: Vec<(SchemaReference, Option<SubgraphDeploymentId>)>,
+    /// Timestamp for the last merge
+    last_refresh: Instant,
```

Comment:
This requires more explanation; the comment at the top says that these entries are immutable. At the very least, there should be something about what happens in between refreshes when data underneath us changes. What happens when queries run while the cache is out-of-date? Is it a problem that different query nodes could have more or less out-of-date caches?